make/photon/prepare/migrations/__init__.py

```diff
@@ -2,4 +2,4 @@
 
 MIGRATION_BASE_DIR = os.path.dirname(__file__)
 
-accept_versions = {'1.9.0', '1.10.0', '2.0.0', '2.1.0'}
\ No newline at end of file
+accept_versions = {'1.9.0', '1.10.0', '2.0.0', '2.1.0', '2.2.0'}
```

Comment:
I don't think we need to keep adding versions as we only support n-2 upgrading path?

Reply:
This is the input config's version list which our prepare script can handle. Even we only maintain n-2 versions, we shouldn't block earlier versions to upgrade.

Reply:
Fair enough.